examples/asr/conf/fastconformer/fast-conformer_transducer_bpe.yaml

```diff
@@ -145,6 +145,7 @@ model:
     xscaling: true # scales up the input embeddings by sqrt(d_model)
     untie_biases: true # unties the biases of the TransformerXL layers
     pos_emb_max_len: 5000
+    use_pytorch_sdpa: true
```

Comment:
Please add one line comment on what it does in the config.

Reply:
Also let's add it option to the ctc config too.

Reply:
is it necessary? it's already turned on by default in `__init__`

Reply:
Let's have it here at least to make it exposed to users otherwise it would be hidden.

Reply:
done